src/App.css

```diff
@@ -10,43 +12,160 @@
   font-size: 2rem;
 }
 
-.Title {
-  margin: 2rem;
-}
-
 .Header {
   display: flex;
   flex-direction: row;
+  border-bottom: 1px solid #d8d8d8;
+}
+
+.Title {
+  margin: 2rem;
+  font-size: 1.8rem;
+  color: #0290ff;
 }
 
 .NavBar {
   display: flex;
-  justify-content: space-between;
   width: 100vw;
+  background-color: white;
 }
 
 .NavBar ul {
   list-style-type: none;
   display: flex;
   margin: 2rem;
+  color: #121212;
+  font-size: 0.8rem;
+  text-transform: uppercase;
+  align-items: center;
 }
 
 .NavBar li {
-  padding: 10px;
+  padding-right: 2rem;
 }
 
 .bookContainer {
   display: grid;
-  grid-template-columns: 2fr 1fr 1fr;
+  grid-template-columns: 3fr 2fr 2fr;
   margin: 3rem;
+  padding: 1rem 2rem;
   align-items: center;
+  background-color: white;
+  border: 1px solid #d8d8d8;
+  border-radius: 0.3rem;
+}
+
+.genre {
+  font-weight: bold;
+  font-size: 1rem;
+  color: #999;
+}
+
+.bookTitle {
+  font-family: 'Roboto Slab', serif;
+  font-weight: bold;
+  color: #121212;
+  font-size: 1.37rem;
+}
+
+.bookAuthor {
+  font-family: 'Roboto Slab', serif;
+  color: #0290ff;
+}
+
+.remove {
+  background-color: transparent;
+  border: none;
+  font-family: 'Roboto Slab', serif;
+  font-size: 1rem;
+  color: #0290ff;
+  padding: 0;
+}
+
+.remove:hover {
+  cursor: pointer;
 }
 
 .addForm {
   margin: 3rem;
   flex-direction: column;
 }
 
-.btn {
+.graphContainer {
+  display: flex;
+  align-items: center;
+}
+
+.percentageContainer {
+  width: 70px;
+  height: 70px;
+  border-radius: 3rem;
+  border: 5px solid #3487e7;
+  border-right: 5px solid #ddd;
+  transform: rotate(225deg);
+  margin: 0 2rem;
+}
+
+.percentage {
+  font-size: 2rem;
+}
+
+.addNew {
+  margin: 3rem;
+  color: #888;
+  text-transform: uppercase;
+}
+
+form input {
+  font-size: 1rem;
+  color: #c4c4c4;
+  border: 1px solid #c4c4c4;
+  border-radius: 4px;
+  margin-right: 0.3rem;
+  height: 2rem;
+  width: 20rem;
+}
+
+::placeholder {
+  font-family: 'Montserrat', sans-serif;
+  color: #c4c4c4;
+}
+
+select {
+  border: 1px solid #c4c4c4;
+  border-radius: 4px;
+  font-size: 1rem;
+  color: #c4c4c4;
+  height: 2.2rem;
+  width: 10rem;
+}
+
+.add {
+  background-color: #3487e7;
+  color: white;
+  font-family: 'Roboto Slab', serif;
+  font-weight: bold;
+  height: 2.2rem;
+  border: none;
+  border-radius: 4px;
+  margin-left: 1rem;
   width: 4rem;
 }
+
+.btn:hover {
+  cursor: pointer;
+}
+
+.progressContainer {
+  font-family: 'Roboto Slab', serif;
+}
+
+.progressBtn {
+  background-color: #3487e7;
+  color: white;
+  font-family: 'Roboto Slab', serif;
+  height: 2.2rem;
+  border: none;
+  border-radius: 4px;
+  padding: 0 2rem;
+}
```

Comment:
- [ ] <b>REQUIRED:</b> There are still few things we need to fix to make our app look more like the provided design. Kindly review the following issues and make sure to fix them.

| Original      | Yours |
  | ----------- | ----------- |
| ![80047697-0c3f6280-8506-11ea-9416-a4f1800e2329](https://user-images.githubusercontent.com/45217364/130871998-88863e2f-c823-4b7b-a5c4-20162973ff21.png) | ![scrnli_2_27_2022_7-27-27 AM](https://user-images.githubusercontent.com/45217364/155869512-427bc795-78ad-4a04-b3ca-f689be8f70ba.png) |

  - The app is not full screen. it is more like a card in the middle of the screen.
  - The create new form should fit in the content so that its width is the same as the list items and on the same vertical alignment.
  - On the top right corner, there must be a profile icon.